include/eosio/check.hpp

```diff
@@ -20,6 +20,7 @@ void eosio_assert_code(uint32_t, uint64_t);
 }
 #else
 #include <stdexcept>
+#include <stdint.h>
```

Comment:
Can you just move `#include <cstdint>` out of the `#ifdef` ?

Reply:
I considered that, but there is no usage of `std` in here, so `<stdint.h>` seemed the appropriate choice. And I wasn't sure the ramifications of doing something different with abieos' cdt usage. This just seemed like the most conservative choice to simply add a missing include -- not refactor code at all.